spec/models/case/base_spec.rb

```diff
@@ -26,6 +26,7 @@
 #  type                 :string
 #  appeal_outcome_id    :integer
 #  dirty                :boolean          default(FALSE)
+#  user_id              :integer          not null, default(-100), foreign key
```

Comment:
How does this get added to the header?

Reply:
Manually? It's all horribly out of date...